src/test/java/com/revature/controllers/SwaggerfileControllerTest.java

```diff
@@ -0,0 +1,24 @@
+package com.revature.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.revature.docutest.DocutestApplication;
+
+@SpringBootTest(classes = DocutestApplication.class)
+@ExtendWith(SpringExtension.class)
+public class SwaggerfileControllerTest {
+    
+    @Autowired
+    private ObjectMapper om;
+    
+    @Test
+    public void test() {
+        System.out.println(om);
```

Comment:
This test should potentially do something, but I understand that this PR is only a Draft.